parquet/README.md

```diff
@@ -1,33 +1,33 @@
 # Parquet Sample data
 
-This folder contains sample files in various format
+This folder contains sample parquet files that work within the Deephaven console. 
```

Comment:
The repo should be divided up by data sets, not by file types.  Doing it by file types is equivalent to having a software module named `Utils`.  It ends up being an invitation to create a giant mess that never gets cleaned up, since every developer says "my function is a util".  This comes from many prior experiences.

See for example:  https://breadcrumbscollector.tech/stop-naming-your-python-modules-utils/